packages/core/src/components/Autocomplete/index.tsx

```diff
@@ -219,6 +222,10 @@ export default class Autocomplete<T extends Item> extends React.Component<Props<
       this.props.onFocus(event);
     }
 
+    if (this.props.loadItemsOnFocus) {
+      this.loadItems('');
```

Comment:
Maybe this should be the current value and not an empty string?

Reply:
good call when there's a value already else results would be wrong 👍